rules/microsoft-365/credential_access_user_excessive_sso_logon_errors.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/05/17"
+maturity = "production"
+updated_date = "2021/05/17"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies accounts with a high number of single sign-on (SSO) logon errors. Excessive logon errors may indicate an
+attempt to brute force a password or single sign-on token.
```

Comment:
Does an invalid or expired SSO imply excessive logon errors? Maybe we should re-word this to be more specific to the SSO artifact, which we could still imply may be an attempt to brute force or the result of a phish or some other attack